internal/testutil/testutil.go

```diff
@@ -13,6 +13,8 @@ import (
 	"os"
 	"path/filepath"
 	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
```

Comment:
Please delete line.

Reply:
The blank line or the import? Thanks

Reply:
I cant see the error :(

Reply:
The import (and the blank line, but `gofmt`/`goimports` should take care of that).

The import is for the package it's in (kind of like `package foo` has `import foo`, if that makes sense), so it causes a cycle right now:

```
# github.com/GoogleCloudPlatform/golang-samples/spanner/spanner_snippets
import cycle not allowed
package github.com/GoogleCloudPlatform/golang-samples/spanner/spanner_snippets (test)
	imports github.com/GoogleCloudPlatform/golang-samples/internal/testutil
	imports github.com/GoogleCloudPlatform/golang-samples/internal/testutil
FAIL	github.com/GoogleCloudPlatform/golang-samples/spanner/spanner_snippets [setup failed]
```

Reply:
Here is how to run the tests (they will skip right now without Spanner setup, but it should still show you the compilation failure):

```
cd $GOPATH/src/github.com/GoogleCloudPlatform/golang-samples/spanner/spanner_snippets
go test -v ./...
```

(Note: the default `GOPATH` is `~/go`. You can see the current value with `go env GOPATH`. [CONTRIBUTING.md](https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md) has some guidance for how to setup your local environment.)